### PR TITLE
language changes related to teams

### PR DESCRIPTION
**Pull request by karlaturcios**
Could someone take a look at this draft branch? I have a minor rails syntax error on a hint I'm adding to the group label.

### CODE REVIEW

File: app/views/groups/_form.html.erb [hunk 1]
```diff
@@ -18,13 +18,14 @@
                                  :id, :email,
                                  collection_wrapper_tag: :ul,
                                  collection_wrapper_class: 'usa-input-list',
-                                 item_wrapper_tag: :li) do |b| %>
+                                 item_wrapper_tag: :li,
+                                 hint: "Select members to add to this team.".html_safe) do |b| %>
```
Comment (jmhooper) [inline app/views/groups/_form.html.erb]: I was looking at this and it looks like sneaking it in here doesn't actually display a hint. I think you need to do something like this:

```ruby
<%= form.collection_check_boxes( :user_ids,
                                current_user.admin? ? User.all.sorted : User.select { |u| u.groups.include?(@group) },
                                :id, :email,
                                collection_wrapper_tag: :ul,
                                collection_wrapper_class: 'usa-input-list',
                                item_wrapper_tag: :li) do |b| %>
      <%=  b.check_box(class: "usa-checkbox__input", disabled: b.object == current_user && !current_user.admin?) %>
      <%=  b.label(class: "usa-checkbox__label")%>
  <% end %>
  <%= form.hint "Select members to add to this team." %>
```

Comment (jennywagg) [reply app/views/groups/_form.html.erb]: Oh good catch, I wasn't careful about what I was moving the hint tag onto

**Comment by karlaturcios** [comment]
Why: The new label for Groups is now Teams so content has been revised in the app 

How:

Update labels in buttons, navigation, forms
Update form hints

**Comment by karlaturcios** [comment]
Thanks for the review @jmhooper

**Pull request merged**